overlord/fdestate/conflict.go

```diff
@@ -65,3 +71,78 @@ func checkFDEChangeConflict(st *state.State) error {
 	}
 	return nil
 }
+
+func dbxUpdateAffectedSnaps(t *state.Task) ([]string, error) {
+	// TODO:FDEM: check if we have sealed keys at all
+
+	// DBX updates cause a reseal, so any snaps which are either directly
+	// measured or their content is measured during the boot will count as
+	// affected
+
+	// XXX this effectively blocks updates of gadget, kernel & base until the
+	// change completes
+
+	return fdeRelevantSnaps(t.State())
+}
+
+func checkDBXChangeConflicts(st *state.State) error {
+	// TODO:FDEM: check if we have sealed keys at all
```

Comment:
maybe the comment could mentioned that it's a finer grained conflict check that can produce more useful error when exercising DBX related APIs, but should be used in combination with checkFDEChangeConflict

Reply:
updated the doc comment, thanks for the suggestion